src/app/book/[id]/page.module.css

```
@@ -0,0 +1,92 @@
+.each-book-container {
+  .top-section {
+    display: grid;
+    grid-template-columns: auto 1fr;
+    margin: 20px 10px 40px 10px;
```

واقعاً نیازی به این marginها بود؟
اولاً marginهای چپ و راست باعث شده Alignment رعایت نشه پس باید حذف بشن
ثانیاً marginهای بالا و پایین میتونستن یه مقدار مساوی داشته باشن

```suggestion
    margin-block: 2rem;
```